7/paul-battley-captions.srt

```diff
@@ -2922,7 +2922,7 @@ So what I might say is...
 
 687
 01:00:35,200 --> 01:00:37,168
-inaudible
+First of all that's.. that's..
```

Comment:
I'd be ok with dropping the duplicate "that's.." here is it doesn't add much, but no big deal.